irc3/plugins/command.py

```diff
@@ -302,16 +302,18 @@ def do_command(self, predicates, meth, client, target, data=None, **kw):
             data = self.split_command(
                 data, use_shlex=predicates.get('use_shlex', True))
         except ValueError as e:
-            self.context.privmsg(to, 'Invalid arguments: {}.'.format(e))
-            return
+            if not predicates.get('quiet', False):
+                self.context.privmsg(to, 'Invalid arguments: {}.'.format(e))
+                return
```

Comment:
`return` should not be indented: you want to exit regardless of being silent/verbose. If your test below didn't catch this typo, please update your test to verify it did return early.

Reply:
Thanks for the catch. In this case though, if `quiet=True` it would return late? I don't think the test covers this line specifically because it depends on `use_shlex`. I'll look into it.